README.md

```diff
@@ -6,41 +6,126 @@ Transform FHIR to dataset for ML applications
 
 This project is still under development.
 
-This repo is a POC allowing to make a query (close to SQL format) on a FHIR API and to retrieve tabular data.
+This repo is a POC allowing to make a SQL query on a FHIR API and to retrieve tabular data. 
 
-The request that FHIR2Dataset performs on a FHIR API is specified in a configuration file of this [form](examples/config_template.json).
-The purpose of this file is to perform a query whose result will be identical to the next SQL query:
+## Installation
+
+After cloning this repository, you can install the required dependencies
+
+```
+pip install -r requirements.txt
+npm install --prefix ./fhir2dataset/metadata
+```
+
+For usage, refer to this [turorial](https://htmlpreview.github.io/?https://github.com/arkhn/FHIR2Dataset/blob/query_tests/examples/tutorial.html) and then this [Jupyer Notebook](examples/example.ipynb)
+
+
+## Getting started
+
+Two possible ways to enter the query : as a SQL query or as a JSON config file
+
+**SQL query as entry**
+
+```
+import pandas as pd
```

Comment:
I wonder if we can do without the `import pandas`, what do you think?

Reply:
Yes it is true thanks ! In the demo tutorial we can delete it too, don't you think ?

Reply:
Yes!  Great point, we can delete it, it's useless :smile: